omod/src/test/java/org/openmrs/module/referenceapplication/page/controller/HomePageControllerTest.java

```diff
@@ -42,12 +38,9 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
Try not to use wildcard import. Ref: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Imports